Corona-Warn-App/src/main/java/de/rki/coronawarnapp/tracing/states/TracingState.kt

```diff
@@ -144,12 +131,11 @@ data class LowRisk(
         )
     }
 
-    fun getRiskActiveTracingDaysInRetentionPeriod(context: Context): String =
-        if (activeTracingDays < TimeVariables.getDefaultRetentionPeriodInDays()) {
-            context.getString(R.string.risk_card_body_saved_days).format(activeTracingDays)
-        } else {
-            context.getString(R.string.risk_card_body_saved_days_full)
-        }
+    fun getDaysSinceInstall(context: Context): String =
+        context.getString(R.string.risk_card_body_days_since_installation)
+            .format(daysSinceInstallation.roundUpMsToDays())
```

Comment:
maybe  re-name it to `timeSinceInstalltion` or 
`val daysSinceInstallation: Long get() = (System.currentTimeMillis() - installTime).roundUpMsToDays()` ?